src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -66,7 +82,7 @@ public boolean equals(Object other) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(this.organization, this.repoName, this.branch);
+        return Objects.hash(location);
```

Comment:
perhaps hash displayname since its supposed to be unique? location can be different e.g. http and https etc.